Dockerfile

```diff
@@ -22,9 +22,10 @@ RUN rm /tmp/cuchem_rapids_0.17.yml
 RUN source activate cuchem && python3 -m ipykernel install --user --name=cuchem
 RUN echo "source activate cuchem" > /etc/bash.bashrc
 
-COPY launch.sh /opt/nvidia/cheminfomatics/
-COPY *.py /opt/nvidia/cheminfomatics/
-COPY nbs/*.ipynb /opt/nvidia/cheminfomatics/
+RUN mkdir -p /opt/nvidia/ \
```

Comment:
Perhaps this was known, but the still seems to be missing most of the library files:

```
# ls /opt/nvidia/cheminfomatics/
Dockerfile  LICENSE  README.md  chemvisualize.py  demo.ipynb  icla.pdf  launch.sh  screenshot.jpg  startdash.py
```